src/redis/t_zset.c

```diff
@@ -1123,6 +1123,19 @@ unsigned char *zzlDeleteRangeByRank(unsigned char *zl, unsigned int start, unsig
  * Common sorted set API
  *----------------------------------------------------------------------------*/
 
+zset* zsetCreate(void) {
+    zset *zs = zmalloc(sizeof(*zs));
+    zs->dict = dictCreate(&zsetDictType);
+    zs->zsl = zslCreate();
+    return zs;
+}
+
+void freeZset(zset *zs) {
```

Comment:
It seems like other functions in this file _start_ with `zset` rather than end with it, perhaps do the same here?